packages/webdriverio/src/commands/element/isStable.ts

```diff
@@ -0,0 +1,72 @@
+import { getBrowserObject } from '../../utils/index.js'
+
+/**
+ *
+ * Will return true when stable (in animation) or when unstable (not in animation)
+ *
+ * <example>
+    :index.html
+    <head>
+    <style>
+        div {
+        width: 200px;
+        height: 200px;
+        background-color: red;
+        animation: 3s 0s alternate slidein;
+        }
+        @keyframes slidein {
+        from {
+            margin-left: 100%;
+            width: 300%;
+        }
+
+        to {
+            margin-left: 0%;
+            width: 100%;
+        }
+        }
+    </style>
+    </head>
+    <body>
+    <div></div>
+    </body>
+
+    :isStable.js
+    it('should detect if an element is stable', async () => {
+        let element = await $('#has-animation');
+        console.log(await element.isStable()); // outputs: true
+
+        element = await $('#has-no-animation')
+        console.log(await element.isStable()); // outputs: false
+    });
+ * </example>
+ *
+ * @alias element.isStable
+ * @return {Boolean} true if element is stable, false if unstable
+ * @type state
+ *
+ */
+export async function isStable (this: WebdriverIO.Element) {
+    const browser = getBrowserObject(this)
+    return await browser.executeAsync((elem, done) => {
```

Comment:
Let's move the function into an external script in `packages/webdriverio/src/scripts` and give it a name so we don't print the whole function content within the wdio logs

Reply:
having these scripts in that directory will help us later on compile them to a different target if needed, e.g. to ensure that we support older browser.